plugin.php

```diff
@@ -39,9 +40,12 @@ function genesis_simple_share_init() {
 		require_once( GENESIS_SIMPLE_SHARE_LIB . 'admin.php' );
 		require_once( GENESIS_SIMPLE_SHARE_LIB . 'post-meta.php' );
 	}
-	else
+	else{
 		require_once( GENESIS_SIMPLE_SHARE_LIB . 'front-end.php' );
+	}
 
 	//require_once( GENESIS_SIMPLE_SHArE_LIB . 'functions.php' );
 
 }
+
+require_once( GENESIS_SIMPLE_SHARE_LIB . 'ajax.php' );
```

Comment:
Should be required conditionally.
~~~php
if ( defined( 'DOING_AJAX' ) && DOING_AJAX ) {
	require_once( GENESIS_SIMPLE_SHARE_LIB . 'ajax.php' );
}
~~~